docs/source/index.rst

```diff
@@ -42,6 +46,7 @@ tutorial section.
    start/intro-to-api
    start/how-to-explore-capella-mm
    start/developing-docs
+   start/declarative
```

Comment:
Hm, I haven't really thought about where best to put it, I just added it at the end.

I think it would definitely make sense to keep the metamodel exploration and docs development together, because they're both targeted at contributors to the library, whereas the new page targets users. So the first one would make the most sense, where it's right after the introduction into the general API.